lib/engine.js

```diff
@@ -397,10 +397,11 @@ module.exports = function (s, conf) {
             trade_balance = n(s.balance.currency).divide(100).multiply(buy_pct)
             tradeable_balance = n(s.balance.currency).divide(100 + fee).multiply(buy_pct)
             expected_fee = n(trade_balance).subtract(tradeable_balance).format('0.00000000', Math.ceil) // round up as the exchange will too
+            console.log(`s.product.asset_increment: ${s.product.asset_increment}`)
```

Comment:
Use our pseudo msg() instead of console.log()

Reply:
I'll just delete that line